virtual_rainforest/core/config.py

```diff
@@ -1,10 +1,42 @@
-"""The `core.config` module.
-
-The `core.config` module is used to read in the various configuration files, validate
-their contents, and then configure a ready to run instance of the virtual rainforest
-model.
+"""API documentation for the :mod:`core.config` module.
+************************************************** # noqa: D205
+
+The :mod:`core.config` module is used to read in the various configuration files,
+validate their contents, and then configure a ready to run instance of the virtual
+rainforest model. The basic details of how this system is used can be found
+:ref:`here<virtual_rainforest/core/config:the configuration module>`.
+
+When a new module is defined a ``JSON`` file should be written, which includes the
+expected configuration tags, their expected types, any value constraints, and where
+appropriate default values. This schema should be saved in the folder of the module that
```

Comment:
there is a verb missing 'where appropriate default values??'

Reply:
Yeah I tried to cram too much info into one list there, I'll separate it out a bit